README.md

```diff
@@ -508,8 +508,7 @@ paste the following default values
     mscs-maps-location=/opt/mscs/maps
     mscs-maps-url=my.minecraftserver.com
     
-* `mscs-overviewer-bin`: The overviewer file. If you did the default Overviewer installation, 
-   leave this untouched.
+* `mscs-overviewer-bin`: This is the location for the Overviewer binary.                                                                                                                       If you manually installed Overviewer to another location, you                can enter the location here.
```

Comment:
Weird formatting here -- extra white space.  When viewed as a web page it's not an issue, but it looks odd when viewed as a text file.